### PR TITLE
[timeseries] Bump optimum dependency ranges

### PR DESCRIPTION
**Pull request by canerturkmen**
*Issue #, if available:*

#4044

*Description of changes:*

Update optimum and optimum-intel dependencies.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by yinweisu** [comment]
| Previous CI Run | Current CI Run |
| :---: | :---: |

**Comment by yinweisu** [comment]
| Previous CI Run | Current CI Run |
| :---: | :---: |

**Comment by yinweisu** [comment]
| Previous CI Run | Current CI Run |
| :---: | :---: |

File: timeseries/setup.py [hunk 1]
```diff
@@ -56,10 +56,11 @@
         "black~=23.0",
     ],
     "chronos-openvino": [  # for faster CPU inference in pretrained models with OpenVINO
-        "optimum[openvino,nncf]>=1.17,<1.18",
+        "optimum-intel[openvino,nncf]>=1.16,<1.17",
+        "optimum[openvino,nncf]>=1.18,<1.19",
```
Comment (shchur) [inline timeseries/setup.py:60]: Does this place any restrictions on the `pandas` version range?

**Review by shchur — APPROVED** [review]

File: timeseries/setup.py (hunk 1 above)
Comment (canerturkmen) [reply timeseries/setup.py:60]: No, doesn't change any of the pandas dependency constraints for now. will start allowing pandas 2.2 once nncf releases 2.9.

**Pull request merged**